ansible/requirements.txt

```diff
@@ -47,7 +47,7 @@ PyNaCl==1.3.0
 pytest==4.1.1
 python-dateutil==2.7.5
 python-gilt==1.2.1
-PyYAML==3.13
+PyYAML>=4.2b1
```

Comment:
Avoid using open-ended ranges. This runs the risk of new versions of the library breaking the installation.